src/sensor/ping.cpp

```diff
@@ -497,7 +497,7 @@ void Ping::checkNewFirmwareInGitHubPayload(const QJsonDocument& jsonDocument)
     float lastVersionAvailable = 0.0;
 
     auto filesPayload = jsonDocument.array();
-    for(const QJsonValue& filePayload : filesPayload) {
+    for(const QJsonValue filePayload : filesPayload) {
```

Comment:
I believe that you want to use QJsonValueRef here since you are moving away from reference.

Reply:
Probably. I actually would use nlohmann json for this, seems saner. but right now we are not depending on it and I don't want to add new dependencies.